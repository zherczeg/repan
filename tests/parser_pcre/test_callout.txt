# Test callout blocks

/.(?C).(?C0).(?C5).(?C255)./u
    print

/(?C256)/u
    print

/(?C-1)/u
    print

/(?C4/u
    print

/.(?C`ab``cd`).(?C``).(?C'''').(?C"").(?C^^).(?C%%).(?C##).(?C$$).(?C{a}}b})./u
    print

/(?C"a"/u
    print

/(?C"a""/u
    print
