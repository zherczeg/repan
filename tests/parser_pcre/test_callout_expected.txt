# Test callout blocks

/.(?C).(?C0).(?C5).(?C255)./u
    print
    /.(?C0).(?C0).(?C5).(?C255)./

/(?C256)/u
    print
    Pattern parse failed: Callout number must be between 0 and 255
        /[ HERE ]>>(?C256)/

/(?C-1)/u
    print
    Pattern parse failed: Invalid (?C...) callout block
        /[ HERE ]>>(?C-1)/

/(?C4/u
    print
    Pattern parse failed: A right bracket is expected
        /(?C4[ HERE ]>>/

/.(?C`ab``cd`).(?C``).(?C'''').(?C"").(?C^^).(?C%%).(?C##).(?C$$).(?C{a}}b})./u
    print
    /.(?C`ab``cd`).(?C``).(?C'''').(?C"").(?C^^).(?C%%).(?C##).(?C$$).(?C{a}}b})./

/(?C"a"/u
    print
    Pattern parse failed: A right bracket is expected
        /(?C"a"[ HERE ]>>/

/(?C"a""/u
    print
    Pattern parse failed: Invalid (?C...) callout block
        /[ HERE ]>>(?C"a""/
